docs/meshstack.openshift.index.md

```diff
@@ -296,3 +296,40 @@ For example if you plan to use custom roles named `my-custom-view` and `my-custo
   - my-custom-edit
 # ...
 ```
+
+### Tagging configuration
+
+Openshift supports meshStack's powerful multi-cloud [tagging system](./meshstack.metadata-tags.md#replicate-tags-to-cloud-platforms).
+To enable the Openshift integration to automatically replicate tags from meshStack into the cloud platform, the following configuration must be active.
+
+Additionally, it is possible to replicate any [extra metadata](./meshstack.metadata-tags.md#extra-metadata) tags that are coming from meshStack, e.g. the name of the payment method or the name of the meshProject.
+Use the `tag-mappers` configuration part for this mechanism.
+
+<!--DOCUSAURUS_CODE_TABS-->
+<!--Dhall Type-->
+```dhall
+let OpenshiftPlatformCoreConfiguration =
+    {-
+
+      tenantTags:
+        Configures how to map tags coming from meshStack to project labels in Openshift. For more information please look into
+        the TenantTags type documentation. Please make sure to pick a valid DNS namespace-prefix that ends with a slash!
```

Comment:
t: Maybe we should guard against this mistake in code.

Reply:
Can we make some kind of validator with Dhall that enforces this behavior?

Reply:
We will soon have this configuration configurable in the UI. I think that is the perfect opportunity for this.